scss/atoms/typography/_typography.scss

```diff
@@ -12,10 +12,10 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( max-width: $viewport-sm ) {
```

Comment:
I don't really like the `sm` abbreviation as new contributors are never going to know what this is.